src/components/Broadcasts/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Broadcast } from 'src/components/Broadcast';
+import { Props } from '../BroadcastStatus';
```

Comment:
import するのはいったんこのままでいいですが、以下の修正をお願いします 🙏 
- 以下のようにしてみてください ( `type import` と言います )
```suggestion
import  type { Props } from '../BroadcastStatus';
```
- 命名が Props のままだとなんの型定義なのかわからないので、名前を変えましょう!
  - `as` を使って別名称で import できます
```suggestion
import type { Props as Hoge } from '../BroadcastStatus';
```